test/functionalTest/cases/1875_NGSIv2_notification_formats/NGSIv2_notification_formats.test

```diff
@@ -0,0 +1,509 @@
+                        "type": "none", 
+                        "value": 1
+                    }, 
+                    {
+                        "name": "A2", 
+                        "type": "none", 
+                        "value": 2
+                    }
+                ], 
+                "id": "E1", 
+                "isPattern": "false", 
+                "type": "none"
+            }, 
+            "statusCode": {
+                "code": "200", 
+                "reasonPhrase": "OK"
+            }
+        }
+    ], 
+    "originator": "localhost",
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+POST http://localhost:REGEX(\d+)/notify
+Content-Length: 170
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Host: localhost:REGEX(\d+)
+Accept: application/json
+X-Ngsiv2-Attrsformat: normalized
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
+        {
+            "A1": {
+                "metadata": {}, 
+                "type": "none", 
+                "value": 1
+            }, 
+            "A3": {
+                "metadata": {}, 
+                "type": "none", 
+                "value": 3
+            }, 
+            "id": "E1", 
+            "type": "none"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+POST http://localhost:REGEX(\d+)/notify
+Content-Length: 170
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Host: localhost:REGEX(\d+)
+Accept: application/json
+X-Ngsiv2-Attrsformat: normalized
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
+        {
+            "A1": {
+                "metadata": {}, 
+                "type": "none", 
+                "value": 1
+            }, 
+            "A4": {
+                "metadata": {}, 
+                "type": "none", 
+                "value": 4
+            }, 
+            "id": "E1", 
+            "type": "none"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+POST http://localhost:REGEX(\d+)/notify
+Content-Length: 94
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Host: localhost:REGEX(\d+)
+Accept: application/json
+X-Ngsiv2-Attrsformat: keyvalues
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
+        {
+            "A1": 1, 
+            "A5": 5, 
+            "id": "E1", 
+            "type": "none"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+POST http://localhost:REGEX(\d+)/notify
+Content-Length: 60
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Host: localhost:REGEX(\d+)
+Accept: application/json
+X-Ngsiv2-Attrsformat: values
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
```

Comment:
This test bring me something to memory... From NGSIv2 spec (section describing subscription payload):

> attrs [at notification]: List of attributes to be included in the notification message. If not specified (or empty), all attributes are included in the notification. It also defines the order in which attributes will appear in notifications when attrsFormat value is used (see "Notification messages" section).

An additional .test covering this case should be included (maybe modifications in the code are needed also), both the three NGSIv1 modes: normalized, keyValues and values (it is specially important in the third case).